test/integration/react-streaming-and-server-components/test/rsc.js

```diff
@@ -120,10 +120,13 @@ export default function (context, { runtime, env }) {
   })
 
   it('should handle streaming server components correctly', async () => {
+    const temp = process.env.BROWSER_NAME
+    process.env.BROWSER_NAME = 'firefox'
```

Comment:
```suggestion
```
This shouldn't be needed as the env should ideally be set when running `jest` or it will always be forced to run in firefox here